src/elections/services/elections.service.ts

```diff
@@ -82,11 +109,11 @@ export class ElectionsService {
     return projectedTurnout
       ? {
           ...this.calculateRaceTargetMetrics(projectedTurnout.projectedTurnout),
-          projectedTurnout: projectedTurnout.projectedTurnout,
           source: P2VSource.ElectionApi,
           electionType: projectedTurnout.L2DistrictType,
           electionLocation: projectedTurnout.L2DistrictName,
           p2vStatus: P2VStatus.complete,
+          p2vCompleteDate: new Date().toISOString().slice(0, 10),
```

Comment:
Same here.. we have a date formatter for ISO dates.. might be good to just use that.. 

https://github.com/thegoodparty/gp-api/blob/develop/src/shared/util/date.util.ts#L19-L24